src/libxtp/qmpackage.cc

```diff
@@ -164,7 +163,6 @@ void QMPackage::ReorderMOsToXTP(Eigen::MatrixXd& v,
                                 const AOBasis& basis) const {
 
   std::vector<Index> multiplier = getMultiplierVector(basis);
-  MultiplyMOs(v, multiplier);
```

Comment:
It is unclear to my what were doing those `multiplier`s before and why we don't need them anymore.

Reply:
We only have to multiply once in the conversion process, I understood the old routine which did the back and to transformations together incorrectly. I think the multipliers come because of different normalization conditions for the regular harmonics, but I genuinely do not know. Bjoern introduced them because otherwise it would not work.